doc/_static/css/custom.css

```diff
@@ -870,6 +866,10 @@ kbd, .kbd,
     text-align: center;
 }
 
+.rst-content figure img {
```

Comment:
fwiw I am not sure about the "div.figure" selector (so I did not include it in this rule) that seems to be a thing in the default sphinx theme, in addition to "figure" -- afaik we don't ever use the "figure" class and only figure elements. Or maybe that's a thing for Latex/PDF styling? @gmarull maybe you have some thoughts.

Reply:
Ya but I meant these kind of rules confused me: https://github.com/sphinx-doc/sphinx/blob/d3c91f951255c6729a53e38c895ddc0af036b5b9/sphinx/themes/basic/static/basic.css_t#L460
as I couldn't find any instance of elements having a class="figure", only <figure>s, but anyway, thanks for the prompt +1!